openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -121,6 +122,20 @@ def is_microfrontend_enabled_for_user(self):
             is_course_staff=self.original_user_is_staff
         )
 
+    @property
+    def is_learning_sequences_api_enabled(self):
+        """
+        Should the Learning Sequences API be used to load course structure data?
+
+        Courseware views in the Learning MFE need to load course structure data
+        from the backend to display feaures like breadcrumbs, the smart "Next"
+        button, etc. This has been done so far using the Course Blocks API.
+        We are switching the views to instead use the Learning Sequences API,
+        which we expect to be significantly faster. We are putting the transition
+        behind a flag, for now, to allow incremental rollout and comparison testing.
```

Comment:
Nit: Should this docstring (or the waffle flag comments above) say anything about incrementally honoring this flag in courseware calls? We'll have an interim period where this flag will be somewhat misleading - making someone reading the code to think that it *actually* controls the whether the Learning Sequences API is used - when it doesn't yet in all cases.

Reply:
Agreed! I'll call that out better.